src/meta_schedule/feature_extractor/utils.h

```diff
@@ -321,6 +359,34 @@ inline int64_t GetVarStride(const std::vector<MultiIndex>& multi_indices,
   return (result == kNotFound) ? 0 : result;
 }
 
+/*!
+ * \brief Given an array of regions, return the minimal stride of a variable on it
+ * \param regions The regions that the buffer is accessed
+ * \param buffer_stride The stride of the buffer
+ * \param var The variable to be checked
+ * \return The minimal stride of the variable on the regions
+ */
+inline int64_t VarStrideOnRegion(const std::vector<NDIntSet>& regions,
+                                 const std::vector<int64_t>& buffer_stride, const tir::Var& var) {
```

Comment:
Similarly, let's unify the names between `VarStrideOnRegion` and `GetVarStride`, both of which serve similar functionality

Reply:
Addressed :)